test/IcalCommandTest.py

```diff
@@ -80,5 +80,31 @@ def test_help_python32(self):
         self.assertEqual(self.output, "")
         self.assertEqual(self.errors, "icalendar is not supported in this Python version.\n")
 
+class IcalCommandUnicodeTest(CommandTest):
+    def setUp(self):
+        super(IcalCommandUnicodeTest, self).setUp()
+        self.todolist = load_file_to_todolist("test/data/ListCommandUnicodeTest.txt")
+
+    @unittest.skipIf(IS_PYTHON_32, "icalendar is not supported for Python 3.2")
+    def test_ical_unicode(self):
+        def replace_ical_tags(p_text):
+            # replace identifiers with dots, since they're random.
+            result = re.sub(r'\bical:....\b', 'ical:....', p_text)
+            result = re.sub(r'\bUID:....\b', 'UID:....', result)
+
+            return result
+
+        command = IcalCommand([""], self.todolist, self.out, self.error)
+        command.execute()
+
+#        self.assertTrue(self.todolist.is_dirty())
```

Comment:
Of course it should. I've commented it out to test something else, and then forgotten about it. Sorry.